pytorch_to_returnn/torch/nn/functional.py

```diff
@@ -194,22 +194,22 @@ def truediv(x: Tensor, y: Tensor) -> Tensor:
 
 def greater(x: Tensor, y: Tensor) -> Tensor:
   dtype = result_type(x, y)
-  return modules.ComparisonOperator(kind="greater")(cast(x, dtype), cast(y, dtype))
+  return modules.ComparisonOperator(kind="greater").as_returnn_torch_functional()(cast(x, dtype), cast(y, dtype))
```

Comment:
Note that `.as_returnn_torch_functional` is never needed when the module itself already has `is_original_torch_module = False` set. I would avoid it in that case as it makes the code a bit harder to read.

This simply marks that the converter logic should not try to search for a corresponding PyTorch module call when comparing the results.